docs/versioned_docs/version-v2.3/01-index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Arcus Observability"
+layout: default
+slug: /
+sidebar_label: Welcome
+---
+
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.Observability.Correlation?includePreReleases=true)](https://www.nuget.org/packages/Arcus.Observability.Correlation/)
```

Comment:
Should probably be the fixed version here, no?

Reply:
Added to #235